Assets/Global/Scripts/Player/Player.cs

```diff
@@ -1,42 +1,42 @@
 using System;
 using UnityEngine;
+using UnityEngine.Serialization;
 
 public class Player : MonoBehaviour
 {
-    
-    public float jumpforce = 2f;
+    [Header("Settings")]
+    public float jumpForce = 2f;
     public float speed = 5f;
+    public float airBornMovementFactor = 0.5f;
     public int doubleJumps = 1;
     public float glideDrag = 2f;
     
-    [HideInInspector]
-    public BaseState currentState;
-    [HideInInspector] 
-    public int currentJumps = 0;
-    [HideInInspector]
-    public float horizontalInput;
-    [HideInInspector]
-    public float verticalInput;
-    public Rigidbody playerRb;
-    [HideInInspector]
-    public bool isGrounded;
+    [Header("References")]
+    [FormerlySerializedAs("playerRb")] 
+    public Rigidbody rb;
```

Comment:
Renamed `playerRb` to `rb`.
We are already in the player, then if we want to have a rb var for this playerRb. We dont have to actually add `player` to the name since we already know its a player.